apache-camel-mixin/README.md

```diff
@@ -0,0 +1,121 @@
+THis is a fork of the community dashboard that can be found here:
+
+- [Github]([https://link](https://github.com/alainpham/app-archetypes/blob/master/camel-monitoring/dashboards-for-import/apache-camel-micrometer.json))
```

Comment:
Are there any changes to the upstream dashboard in this fork?

Reply:
no there are no changes. I want to keep things in sync. Is there any reccomendation on how to handle this? Should we create a link or something like that?

Reply:
The best choice is to use [jsonnet bundler](https://github.com/jsonnet-bundler/jsonnet-bundler) and vendor your existing dashboard, from it's current location..

```
$ jb init
$ jb install https://github.com/alainpham/app-archetypes/camel-monitoring/dashboards-for-import/
```

Then you can remove the `dashboards` folder, and the copy of the dashboard inside it, and change your `mixin.libsonnet` to point to the vendored directory.

```
{
  grafanaDashboards: {
    'apache-camel-micrometer.json': (import 'dashboards-for-import/apache-camel-micrometer.json'),
  },
}
```

Then, if you make changes to your repository, you can simply update it with jsonnet bundler. `jb update`